common/BUILD.gn

```diff
@@ -88,32 +68,20 @@ source_set("common") {
     ]
 
     public_deps = [
-      "extensions/api",
+      "//brave/components/brave_component_updater/browser",
+      "//brave/common/extensions/api",
+      "//extensions/common:common"
     ]
   }
 
-  deps += [
-    "//base",
-    ":network_constants",
-    ":pref_names",
-    ":shield_exceptions",
-    ":switches",
-    "//brave/chromium_src:common",
-    "//brave/components/resources",
-    "//components/resources",
-    "//extensions/buildflags",
-    "//services/service_manager",
-    "//ui/base",
-  ]
-
   if (!is_ios) {
```

Comment:
isn't it always `!is_ios` for us? Do we need such checks across our `gn` files? @bridiver